about.html

```diff
@@ -1,73 +1,199 @@
 <!doctype html>
 
-<html lang="en">
+
+
+<html lang= "en">
+
+
+
 <head>
-  <meta charset="utf-8">
+    
+    
+    <meta charset= "utf-8">
 
-  <title>Sprint Challenge - About</title>
+    <title> Sprint Challenge - About
+    </title>
 
-  <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
+    <link href= "https://fonts.googleapis.com/css?family=Roboto|Rubik" rel= "stylesheet">
+    <link rel= "stylesheet" href= "css/index.css">
 
-</head>
 
-<body>
-    <div class="container about-page">
-     
-    About
+</head>
 
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
 
 
-    <img src="img/about-plan.png" alt="strategy">
+<body>
+    
+    
+    <header>
+    
     
-    Strategy
+        <div class= "icon">
+            <img src= "img/lambda-black.png"/>
+        </div>
+    
+        <nav class= "links">
+        
+            <div class= "h">
+                <a href= "index.html"> Home
+                </a>
+            </div>
+            
+            <div class= "a">
+                <a href= "about.html"> About
+                </a>
+            </div>
+        
+            <div class= "p">
+                <a href= "#products"> Products
+                </a>
+            </div>
+            <div class= "b">
+                <a href= "#blog"> Blog
+                </a>
+            </div>
+        
+            <div class= "c">
+                <a href= "#contact"> Contact
+                </a>
+            </div>
     
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+        </nav>
 
-    Learn More
 
+    </header>
 
 
-    <img src="img/about-working.png" alt="strategy">
-    
-    How We Work
+    <section>
+  
+
```

Comment:
Always try to fill out the alt="" attribute when using images, that way if the image doesn't load the user can tell what the image is about when they mouse over it.